File: 026_Remove_Duplicates_from_Sorted_Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        tail = 1
        length = len(nums)
        if length < 2:
            return length
        for i in range(1, length):
            if nums[i] != nums[i-1]:
                nums[tail] = nums[i]
                tail += 1
        return tail

File: 026_Remove_Duplicates_from_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();
        if (n < 2) return n;
        int i = 1;
        for (int j = 1; j < n; ++j) {
            if (nums[j] != nums[i - 1])
                nums[i++] = nums[j];
        }
        return i;
    }
};
